src/coreclr-debug/debugConfigurationProvider.ts

```diff
@@ -59,6 +61,42 @@ export class DotnetDebugConfigurationProvider implements vscode.DebugConfigurati
                 return undefined;
             }
         }
+        
+        if (!this.platformInformation.isLinux() && !vscode.env.remoteName && vscode.env.uiKind != vscode.UIKind.Web) 
+        {
+            if(debugConfiguration.checkForDevCert === undefined && debugConfiguration.serverReadyAction && !debugConfiguration.pipeTransport)
+            {
+                debugConfiguration.checkForDevCert = true;
+            }
+
+            if (debugConfiguration.checkForDevCert)
+            {
+                hasDotnetDevCertsHttps(this.options.dotNetCliPaths).then(async (hasDevCert) => {
+                    if(!hasDevCert)
+                    {
+                        const result = await vscode.window.showInformationMessage(
+                            "The selected launch configuration is configured to launch a web browser but no trusted development certificate was found. Create a trusted self-signed certificate?", 
+                            "Don't Ask Again", 'Not Now', 'Yes'
```

Comment:
>"Don't Ask Again", 'Not Now', 'Yes' [](http://example.com/codeflow?start=28&length=35)

Is "Don't Ask Again" the default value? If so, I would reverse the order so "Yes" is the default.

Reply:
On other info messages "don't ask again" is usually the default, but I can see why "Yes" would be preferred.
<img width="400" alt="image" src="https://user-images.githubusercontent.com/119355875/220466938-3e181078-18b9-4d77-97c6-70dee3946c08.png">

Reply:
Changed to 
<img width="433" alt="image" src="https://user-images.githubusercontent.com/119355875/220472011-c2ef54e2-d1fc-4a17-befe-be6f64707715.png">

Reply:
We could use the `detail` property to provide this information: [https://github.com/microsoft/vscode/blob/c83eff40dd2c2115f5fd5e67a6ce304fbc6fdefa/src/vscode-dts/vscode.d.ts#L1974](https://github.com/microsoft/vscode/blob/c83eff40dd2c2115f5fd5e67a6ce304fbc6fdefa/src/vscode-dts/vscode.d.ts#L1974)